api/reviewer_api/services/documentpageflagservice.py

```diff
@@ -6,69 +6,140 @@
 from reviewer_api.services.redactionlayerservice import redactionlayerservice
 from reviewer_api.models.default_method_result import DefaultMethodResult
 
-class documentpageflagservice:
 
-    
+class documentpageflagservice:
     def getpageflags(self, requestid, redactionlayerid):
-        layerids = redactionlayerservice().getmappedredactionlayers({"redactionlayerid": redactionlayerid})
+        layerids = redactionlayerservice().getmappedredactionlayers(
+            {"redactionlayerid": redactionlayerid}
+        )
         return DocumentPageflag.getpageflag_by_request(requestid, layerids)
-    
+
     def getpublicbody(self, requestid):
         return DocumentPageflag.getpublicbody_by_request(requestid)
-    
-    def getdocumentpageflags(self, requestid, redactionlayerid, documentid=None, version=None):
-        layerids = redactionlayerservice().getmappedredactionlayers({"redactionlayerid": redactionlayerid})
-        pageflag  = DocumentPageflag.getpageflag(requestid, documentid, version, layerids)
+
+    def getdocumentpageflags(
+        self, requestid, redactionlayerid, documentid=None, version=None
+    ):
+        layerids = redactionlayerservice().getmappedredactionlayers(
+            {"redactionlayerid": redactionlayerid}
+        )
+        pageflag = DocumentPageflag.getpageflag(
+            requestid, documentid, version, layerids
+        )
         if pageflag not in (None, {}):
             return pageflag["pageflag"], pageflag["attributes"]
         return [], None
-    
-    def removebookmark(self,requestid, redactionlayerid, userinfo):
-        pageflags = self.getpageflags(requestid)
+
+    def removebookmark(self, requestid, redactionlayerid, userinfo):
+        pageflags = self.getpageflags(requestid, redactionlayerid)
```

Comment:
passed "redactionlayerid" as 2nd parameter as it is expected. otherwise getting run time error.